xml/containers-docker-overview.xml

```diff
@@ -10,22 +10,22 @@
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>&deng; overview</title>
  <para>
-  The &deng; is a lightweight virtualization solution to run multiple virtual
+  &deng; is a lightweight virtualization solution to run multiple virtual
   Linux environments (containers) simultaneously on top of a single Linux
   kernel, without a hypervisor. Containers are isolated using Kernel cgroups
   (<xref linkend="vle-docker-cgroup"/>) and
   <xref linkend="vle-docker-namespace"/>.
  </para>
  <para>
-  Full virtualization solutions, such as &xen;, &kvm;, or &libvirt;, are based
+  Full virtualization solutions, such as &xen;, &kvm;, and &libvirt;, are based
```

Comment:
Why `and`? IHMO `or` is correct. Each of these there is a full virtualization solution of its own.

Reply:
Yes, it is, so we want an inclusive conjunction here — but TBH I do not think it makes much difference in this instance.